### DISCUSSION TITLE
Combine 2 png

### DISCUSSION THREAD

**Discussion by valdomero777** [dlemstra/Magick.NET#1053] [Help] [answered]
Hi!
I'm trying to combine 2 images but when I try one image is over to another, and I need to put one at the top and the other below the first image.
I'm using the following code:
 public static void MergeMultipleImages()
        {
            using (var images = new MagickImageCollection())
            {
                // Add the first image
                var first = new MagickImage(SampleFiles.SnakewarePng);
                images.Add(first);

                // Add the second image
                var second = new MagickImage(SampleFiles.SnakewarePng);
                images.Add(second);

                // Create a mosaic from both images
                using (var result = images.Mosaic())
                {
                    // Save the result
                    result.Write(SampleFiles.OutputDirectory + "Mosaic.png");
                }
            }
        }

Thanks in advanced

**Comment by dlemstra** [comment, marked as answer]
It looks like you should be calling `AppendVertically` instead of `Mosaic`.

**Reply by valdomero777** [reply]
Thanks so much, worked excellent!
